src/components/Header/index.jsx

```diff
@@ -58,11 +58,7 @@ const Header = () => {
   const [currentGroup, setCurrentGroup] = useState(
     getLocalStorage("currentGroup") || getLocalStorage("user")?.default_group
   );
-  const history = useHistory();
   const location = useLocation();
-  const handleLogin = () => {
-    history.push(routes.home);
-  };
```

Comment:
Can you please explain the changes done here?

Reply:
From what I can understand, the `handleLogin()` function basically redirects users to the home page when the Log In button on the dropdown is clicked. But when the user is logged out the only page he can access is the home page, so the button serves no purpose. 

Even if it fired the same function as the Log In button on the home page it would've served some purpose but as it stands now it's redundant.

Reply:
What if the user is not logged in and is on a different page, say one of the help page. If they want to login, drop-down provides a simpler path to do it.
What do you think?

Reply:
Agreed, I failed to take that into consideration. I'll revert the changes and just remove the extra divider from the Log In dropdown. So it look like such:
![right_login](https://user-images.githubusercontent.com/57453561/161908033-a5c30978-94d9-4277-bde0-8aec1a905d16.png)